common/static/common/templates/discussion/forum-action-vote.underscore

```diff
@@ -2,7 +2,7 @@
     <span aria-hidden="true" class="display-vote is-hidden" >
       <span class="vote-count"></span>
     </span>
-    <a href="#" class="action-button action-vote" role="checkbox" aria-checked="false">
+    <button class="btn-link action-button action-vote" role="checkbox" aria-checked="false">
```

Comment:
In fact, it is a toggle. When it is clicked, this is the JavaScript that is updating it:
https://github.com/edx/edx-platform/blob/master/common/static/common/js/discussion/views/discussion_content_view.js#L247
```javascript
        DiscussionContentShowView.prototype.updateButtonState = function(selector, checked) {
            var $button;
            $button = this.$(selector);
            $button.toggleClass('is-checked', checked);
            return $button.attr('aria-checked', checked);
        };
```